source/payment_flow/index.html.md.erb

```diff
@@ -120,7 +120,10 @@ they have completed their payment attempt on GOV.UK Pay. This should not be
 JSON-encoded, because backslashes are invalid characters. With test accounts,
 you can [use HTTP for return URLs instead of HTTPS](/security/#https).
 
-You can also provide the user's details if you've collected them earlier in your service, to [prefill fields on the payment page](/optional_features/prefill_user_details).
+You can [prefill the fields on the payment page](/optional_features/prefill_user_details) with the user's details if you collected that information:
```

Comment:
@SandorArpa This pull request should be good to go now. Could you just double-check the facts in this short bullet list are still correct?